site/docs-md/guides/deep-links.md

```diff
@@ -0,0 +1,311 @@
+
+### Add Associated Domain
+
+The final step is to configure the iOS app to recognize incoming links. Open Xcode, then navigate to Signing & Capabilities. Click "+ Capability", then choose Associated Domains. In the Domains entry that appears, edit it using the format `applinks:yourdomain.com`:
+
+![Xcode Associated Domain](/assets/img/docs/guides/deep-links/xcode-associated-domain.png)
+
+## Android Configuration
+
+Android configuration involves creating a site association file and configuring the native app to recognize app links using an intent filter.
+
+### Create Site Association File
+
+The Site Association file requires the SHA256 fingerprint of your Android certificate.
+
+If you don’t have one, create a certificate:
+
+```shell
+keytool -genkey -v -keystore KEY-NAME.keystore -alias ALIAS -keyalg RSA -keysize 2048 -validity 10000
+```
+
+Using your existing (or newly created) Keystore certificate, use the keytool command to list the keystore's details:
+
+```shell
+keytool -list -v -keystore my-release-key.keystore
+```
+
+The printed output will include the SHA256 fingerprint:
+
+![Keytool output](/assets/img/docs/guides/deep-links/keystore-sha256.png)
+
+Next, use Google's [Asset Links tool](https://developers.google.com/digital-asset-links/tools/generator) to create the Site Association file. Fill in the website domain, app package name, and SHA256 fingerprint, then click "Generate statement":
+
+![Android Identifier Config](/assets/img/docs/guides/deep-links/android-config.png)
+
+Copy the JSON output into a new local file under `.well-known/applinks.json`.
+
+```json
+// applinks.json
+[
+  {
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+      "namespace": "android_app",
+      "package_name": "com.netkosoft.beerswift",
+      "sha256_cert_fingerprints": [
+        "43:12:D4:27:D7:C4:14..."
+      ]
+    }
+  }
+]
+```
+
+Deploy the file to your website (hosted on HTTPS), then verify it by clicking the "Test statement" button in the Asset Link tool. If it's configured correctly, a Success message will appear:
+
+> Success! Host [website] grants app deep linking to [app package].
+
+### Add Intent Filter
+
+The final step is to configure the Android app to recognize incoming links. To do so, [add a new Intent Filter](https://developer.android.com/training/app-links/deep-linking#adding-filters) to `AndroidManifest.xml` within the `<activity>` element:
+
+```xml
+<intent-filter android:autoVerify="true">
+    <action android:name="android.intent.action.VIEW" />
+    <category android:name="android.intent.category.DEFAULT" />
+    <category android:name="android.intent.category.BROWSABLE" />
+    <data android:scheme="https" android:host="beerswift.app" />
+</intent-filter>
+```
+
+The complete Activity should look similar to this:
+
+```xml
+<activity
+    android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale"
```

Comment:
just updated this line the other day, so it won't look like this anymore

Reply:
that's ok - why i said "it'll look similar to this:" - because I recognize people's AndroidManifest may be different.